src/process/qemuimg_process_spec.cpp

```diff
@@ -65,13 +66,13 @@ profile %1 flags=(attach_disconnected) {
     QString extra_capabilities;
     QString signal_peer; // who can send kill signal to qemu-img
 
-    if (mu::is_snap())
+    try
     {
         root_dir = mu::snap_dir();
         image_dir = mu::snap_common_dir();         // FIXME - am guessing we work inside this directory
```

Comment:
This is a particular case of my earlier comment. If execution reaches this point and throws, we use a mix of snap and non-snap dirs in the profile.

Reply:
Noted